src/contract/operations.rs

```diff
@@ -288,6 +290,8 @@ pub struct Extension {
     pub assignments: Assignments<GenesisSeal>,
     pub redeemed: Redeemed,
     pub valencies: Valencies,
+    pub witness: ReservedBytes<1, 0>,
+    pub script: ReservedBytes<1, 0>,
```

Comment:
Just out of curiosity, today the size of these fields doesn't need to represent the final sizes (when everything is implemented)?

Reply:
Doesn't. For instance, it can become a bit flag vector, where the first byte will indicate number of bytes for bit flags. Thus, 0 will mean "zero-length bytes", and other values will allow up to 255^8 of individual bit flags

Reply:
Wow, this is very useful and makes a lot of sense.

Thanks for explaining.